qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/settings/QuteValidationSettings.java

```diff
@@ -27,15 +27,25 @@
  */
 public class QuteValidationSettings {
 
+	private static enum Severity {

```

Comment:
When you change the settings on vscode-xml, it should retrigger the validation, but it doesn't work because the retrigger is done by comparing the current validation instance with the new.

You must replace the equals and hasCode like this:

```java
	@Override
	public int hashCode() {
		final int prime = 31;
		int result = 1;
		result = prime * result + (enabled ? 1231 : 1237);
		result = prime * result + ((excluded == null) ? 0 : excluded.hashCode());
		result = prime * result + ((undefinedObject == null) ? 0 : undefinedObject.hashCode());
		return result;
	}

	@Override
	public boolean equals(Object obj) {
		if (this == obj)
			return true;
		if (obj == null)
			return false;
		if (getClass() != obj.getClass())
			return false;
		QuteValidationSettings other = (QuteValidationSettings) obj;
		if (enabled != other.enabled)
			return false;
		if (excluded == null) {
			if (other.excluded != null)
				return false;
		} else if (!excluded.equals(other.excluded))
			return false;
		if (undefinedObject == null) {
			if (other.undefinedObject != null)
				return false;
		} else if (!undefinedObject.equals(other.undefinedObject))
			return false;
		return true;
	}

```